urdf2webots/importer.py

```diff
@@ -73,7 +73,7 @@ def convert2urdf(inFile, outFile=None, normal=False, boxCollision=False,
         packages = re.findall('"package://(.*)"', content)
         if packages:
             packageName = packages[0].split('/')[0]
-            directory = os.path.dirname(inFile)
+            directory = os.path.dirname(os.path.abspath(inFile))
```

Comment:
It seems, that could be simplified:
```suggestion
            directory = inPath
```

Reply:
inPath needs to stay unchanged. It would need to be a copy then. I had lots of troubles in the path with storing one variable in another and trying to keep on static. This commit might not be beautiful, but it works :)

Reply:
`directory = inPath` actually makes a copy, because these variables are strings, not objects. So, it should work exactly as before.